tools/registry.c

```diff
@@ -157,7 +159,8 @@ main( int argc, char *argv[], char *env[] )
       sprintf( fname_wrk,"%s/Registry_irr_diag",dir ) ;
     }
 //  fprintf(stderr,"Registry tmp file = %s\n",fname_wrk);
-    sprintf(command,"/bin/cp %s %s\n",fname_in,fname_wrk);
+    /* we should be able to implement this using posix_spawn */
+    sprintf(command,"/bin/cp \'%s\' \'%s\'\n",fname_in,fname_wrk);
```

Comment:
This will still break if the filename includes apostrophes, but I think that's unlikely enough to not try re-implementing [Python's `shlex.quote`](https://docs.python.org/3/library/shlex.html#shlex.quote).